.project-creation/.skeleton/src/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
General question: Should we have license headers in empty (`__init__.py`) files at all?
(They do not contain anything protect-able under a copyright.)

Reply:
A valid question, but we have used this pattern for some time now. We can discuss this and then remove all headers from init files

Reply:
Yes, can be done later. I've added a discussion topic to our architecture sync minutes